pkg/verifier/notation/notation.go

```diff
@@ -244,8 +245,11 @@ func normalizeVerificationCertsStores(conf *NotationPluginVerifierConfig) error
 		return re.ErrorCodeConfigInvalid.NewError(re.Verifier, conf.Name, re.EmptyLink, nil, "both old VerificationCertStores and new VerificationCertStores are provided, please provide only one", re.HideStackTrace)
 	} else if !isCertStoresByType && isLegacyCertStore {
 		// normalize <store>:<certs> to ca:<store><certs> if no store type is provided
+		legacyCertStore := conf.VerificationCertStores
 		conf.VerificationCertStores = verificationCertStores{
-			trustStoreTypeCA: conf.VerificationCertStores,
+			trustStoreTypeCA: legacyCertStore,
+			// support legacy verfier config format for backward compatibility
+			trustStoreTypeTSA: legacyCertStore,
```

Comment:
I feel for the legacy case, we don't need to support TSA. And making CA and TSA reference the same CertStore is not recommended by notation-spec though it's supported for now.

We could just update the document on the legacy limitation and recommend users to move to current setup.

Reply:
https://github.com/ratify-project/ratify-web/pull/105/files